src/containers/ProfilePage/ProfilePage.js

```diff
@@ -155,8 +155,10 @@ export const DesktopReviews = props => {
 export const CustomUserFields = props => {
   const { publicData, metadata, userFieldConfig } = props;
 
+  const shouldPickUserField = fieldConfig => fieldConfig?.showConfig?.displayInProfile !== false;
   const propsForCustomFields =
-    pickCustomFieldProps(publicData, metadata, userFieldConfig, 'userType') || [];
+    pickCustomFieldProps(publicData, metadata, userFieldConfig, 'userType', shouldPickUserField) ||
+    [];
```

Comment:
is this related to categories?  i'm trying to cherry-pick this PR to an older PR and there are a handful of changes that look unrelated to categories.